crates/swc/tests/fixture/issues-7xxx/7612/input/1.js

```diff
@@ -0,0 +1,16 @@
+const ModalInner = ({ data }) => {
+    return (
+        <Album
+            data={(qcData?.flawImageList || [])?.map(src => ({
+                url: src,
+                describe: qcData?.flawDesc
+            }))}
+            showIndicators
+            useSafeAreaBottom
+            showDescription
+            current={current}
+        >
+            test
+        </Album>
+    )
+}
```

Comment:
Nit: this is enough to trigger the bug:
```suggestion
(qcData?.flawImageList || [])?.map()
```